privacy_bot/find_policies.py

```diff
@@ -131,35 +98,37 @@ def main():
 
     # Fetch data
     if urls:
-        print(">> Privacy Bot",                 file=sys.stderr)
-        print('-' * 80,                         file=sys.stderr)
-        print("Processing %s urls" % len(urls), file=sys.stderr)
-        print("Number of jobs: %s" % jobs,      file=sys.stderr)
-        print('-' * 80,                         file=sys.stderr)
-        print('',                               file=sys.stderr)
+        print('-' * 42, file=sys.stderr)
+        print('{0:35}'.format('Initializing Privacy Bot'), file=sys.stderr)
+        print('-' * 42, file=sys.stderr)
+        print('{0:35}{1:7}'.format('Urls to Process: ', len(urls)), file=sys.stderr)
+        print('{0:35}{1:7}'.format('Number of Jobs: ', jobs), file=sys.stderr)
+        print('-' * 42, file=sys.stderr)
+        print('', file=sys.stderr)
 
         # Find privacy policies
         with futures.ProcessPoolExecutor(jobs) as pool:
             policies = pool.map(get_privacy_policy_url, urls)
 
         # Generate policies_metadata file
-        print('',                       file=sys.stderr)
-        print('Generating policies_metadata file', file=sys.stderr)
-        print('-' * 80, file=sys.stderr)
+        print('', file=sys.stderr)
+        print('{0:35}'.format('Generating policies_metadata file'), file=sys.stderr)
 
         policies_metadata = {}
         for url, policies in zip(urls, policies):
-            print('url:', url, 'policies:', policies)
+            # print('url: ', url, 'policies: ', policies)
             policies_metadata[url] = {
                 "domain": url,
                 "privacy_policies": policies,
                 "locale": "fr-FR"
             }
 
-        with open('found_policies.json', 'w') as output:
+        with open('policies_metadata.json', 'w') as output:
```

Comment:
I'm not sure about the name we should give it. I actually changed it to `found_policies.json`, because it's the output of the script `find_policies`. I'm afraid it can be confusing, because the `fetch_policies` script also outputs metadata on policies.